Octokit/Helpers/ApiUrls.cs

```diff
@@ -1918,5 +1918,933 @@ public static Uri UserAdministrationSuspension(string login)
         {
             return "users/{0}/suspended".FormatUri(login);
         }
+
+        /// <summary>
+        /// Returns the <see cref="Uri"/> that returns the assets specified by the asset id.
+        /// </summary>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <param name="id">The id of the release asset</param>
+        /// <returns>The <see cref="Uri"/> that returns the assets specified by the asset id.</returns>
+        public static Uri Asset(int repositoryId, int id)
+        {
+            return "repositories/{0}/releases/assets/{1}".FormatUri(repositoryId, id);
+        }
+
+        /// <summary>
+        /// Returns the <see cref="Uri"/> that returns all of the assignees to which issues may be assigned.
+        /// </summary>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <returns>The <see cref="Uri"/> that returns all of the assignees to which issues may be assigned.</returns>
+        public static Uri Assignees(int repositoryId)
+        {
+            return "repositories/{0}/assignees".FormatUri(repositoryId);
+        }
+
+        /// <summary>
+        /// Returns the <see cref="Uri"/> for a specific blob.
+        /// </summary>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <returns>The <see cref="Uri"/> for a specific blob.</returns>
+        public static Uri Blob(int repositoryId)
```

Comment:
I agree, I'll fix that. Should I rename public static Uri Blob(string owner, string name) overload too?